packages/gatsby-plugin-mdx/README.md

```diff
@@ -0,0 +1,466 @@
+![Logo](../../img/gatsby-mdx.png)
```

Comment:
this image is missing in this commit ...

see: https://github.com/ChristopherBiscardi/gatsby-mdx/tree/master/img